bot/repository.py

```diff
@@ -102,3 +106,10 @@ def update_bot(self, bot, bot_update: BotUpdate = None):
 
                 session.commit()
                 return bot
+    
+    def delete_bot(self, bot):
+        with self.create_session() as session:

```

Comment:
What is the function of "create session" in this section?

Reply:
In SQLAlchemy, a session is a way to manage the communication between our code and the database. `create_session` simply creates that session object so we can use it.